guide/english/ruby/common-array-methods/index.md

```diff
@@ -185,22 +185,15 @@ end
 ```
 
 #### .map
-The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements.
+The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements:
```

Comment:
same as the suggestion for the previous definition, use back-ticks for enclosing functions and methods to distinguish them from other text